src/components/generic/SettingsContext.tsx

```diff
@@ -0,0 +1,22 @@
+import react from 'react';
+
+// interface for the context's state
+interface SettingsContextType {
```

Comment:
A context is not the best state management approach for Assignment 1. There is no shared state between the components, and it might cause surprising bugs (ex. I set hours in my Countdown timer, but then I see that same hours val in my Stopwatch component)